DietPi-Software | Firefox Sync Server: Fix broken install due to started Rust migration

MichaIng opened the pull request:
**Status**: Ready

**Commit list/description**:
+ DietPi-Software | Firefox Sync Server: Fix broken install due to started Rust migration. We'll keep the pure Python-based server as long as the Rust migration is in process which will probably take until end of the year. Until then it is likely impossible to provide a reliable install method: https://github.com/MichaIng/DietPi/issues/3774#issuecomment-691745477

MichaIng committed:
v6.33

+ DietPi-Software | Firefox Sync Server: Fix broken install due to started Rust migration. We'll keep the pure Python-based server as long as the Rust migration is in process which will probably take until end of the year. Until then it is likely impossible to provide a reliable install method: https://github.com/MichaIng/DietPi/issues/3774#issuecomment-691745477

kinoushe reviewed (dismissed):
No errors during reinstall or from `systemctl status firefox-sync`. Syncing is also working again, so I think this resolves the problem.

MichaIng committed:
v6.33

+ CHANGELOG | Firefox Sync Server: Resolved another issue where the build failed due to transition of the whole project from Python to Rust

MichaIng committed:
v6.33

+ DietPi-Software | Firefox Sync Server: Add link to issue for fixed commit
